sdks/go/pkg/beam/runners/dataflow/dataflowlib/job.go

```diff
@@ -83,6 +87,8 @@ type JobOptions struct {
 	TeardownPolicy string
 }
 
+const replaceIdPlaceholder = "toBeReplaced"
```

Comment:
Nit - making this a constant vs inlining doesn't get us anything since we're not reusing it (and its not meaningful)

Reply:
No longer an issue